server/mscalendar/calendar.go

```diff
@@ -98,3 +103,118 @@ func (m *mscalendar) GetCalendars(user *User) ([]*remote.Calendar, error) {
 
 	return m.client.GetCalendars(user.Remote.ID)
 }
+
+// Poll uses the stored deltaLink in the user's subscription to fetch any events that have changed since the last fetch
+// This should run frequently throughout the day to ensure we process new event updates, in case the Graph notifications are experiencing an outage
+func (m *mscalendar) Poll(user *User) (events []*remote.Event, deltaURL string, err error) {
+	err = m.Filter(
+		withClient,
+		withRemoteUser(user),
+	)
+	if err != nil {
+		return nil, "", err
+	}
+
+	sub, err := m.loadUserSubscription(user.MattermostUserID)
+	if err != nil {
+		return
```

Comment:
Why do we not need to return `nil, "", err` for this err case?

Reply:
It compiles because the return values are named/declared as variables in the function signature. I agree that it should have explicit values in the return statement.